dev-support/create-release/release-util.sh

```diff
@@ -613,6 +613,8 @@ function update_releasenotes {
   else
     mv "${releasenotes}" "${project_dir}/RELEASENOTES.md"
   fi
+  log "${MVN[@]}" spotless:apply -DspotlessFiles='RELEASENOTES*.md,CHANGES*.md'
```

Comment:
Is this the correct place to add this? For 3.x, we do not have CHANGES.md and RELEASENOTES.md in our code base.

Reply:
Hmm. Re-reading this function, it looks like we move the files into project_dir, whether there's managed in git or not. project_dir is the git checkout, isn't it? so then this should be fine for either style or management.

Anyway, let me try it in dry-run on a couple branches and projects and see how it goes.